agent-install/agent-install.sh

```diff
@@ -534,6 +539,9 @@ function install_linux(){
 				if version_gt "$AGENT_VERSION" "$PACKAGE_VERSION" ; then
 					log_notify "Installed agent ${AGENT_VERSION} is newer than the packages ${PACKAGE_VERSION}"
 					if [ ! "$OVERWRITE" = true ] ; then
+            if [ $BATCH_INSTALL ]; then
+              exit 1
```

Comment:
There are a few more places in the code that prompt for interactive response which aren't protected by the BATCH_INSTALL check.

Reply:
Fixed

Reply:
Looks like line 790 is still not protected by a batch install check.

Reply:
Fixed